tests/test_iasimage.py

```diff
@@ -25,11 +26,11 @@ def tearDown(self):
 				pass
 
 	def test_help(self):
-		cmd = ['python', IASIMAGE, '-h']
+		cmd = [ENV, 'python', IASIMAGE, '-h']

```

Comment:
I tested 'setup.py test' on Windows but it didn't work. I think the better way to handle this is to import iasimage as a module to run unit test for it, compared to calling iasimage as a script in the current code.

Reply:
Is Windows support really required? I didn't see tests for Windows in CI, so I assumed that Windows is not a concern.

Nevertheless, I agree, importing the module makes more sense.

Reply:
We are using this tool in our boot loader project https://github.com/slimbootloader/slimbootloader. Many developers and users in firmware space are using Windows environment. Setting up a Windows CI would be a good idea. If you have some resources I am happy to consider / follow up.